### PR TITLE
Write README to scoring directory describing its structure. Closes #5

### LINKED ISSUE #5: Add ReadMe that explains directory structure

### PR DESCRIPTION
**Pull request by rgabbard**

### CODE REVIEW

**Comment by rgabbard** [comment]
@marjorief : Any comments on the README to be written to each scoring directory?

**Comment by rgabbard** [comment]
(click "Files changed" on the pull request to see the file)

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+For those just after the official eval scores, these can be found as:
+
+* the only score in withRealis/argScores/ArgScore.bootstrapped.medians.csv
+* the F1 score in withRealis/linkScores/linkScores.bootstrapped.medians.csv
+
+Bootstrapping was done with 10,000 samples over the corpus.
+
+At the highest level each scoring directory has:
+   * withRealis/ - directory with scores counting realis (the official way)
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: counting realis --> "requiring correct ACTUAL, GENERIC, OTHER" determination for alignment between system and reference"

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+   * noRealis/ - directory with scorings ignoring realis (diagnostic)
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: ignoring realis --->  "ignoring ACTUAL, GENERIC, OTHER determination for alignment between system and reference"

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+   * alignmentFailures/ - directory with information on arguments which failed to
+                             align to the CoreNLP parse used for lenient scoring.
+                             This is purely diagnostic.
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: Is there anything to say about how to use this for diagnostic purposes?

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Each of the first two directories has the following structure:
+    argScores/ - bootstrapped argument scores (official argument metric)
+    linkScores/ - bootstrapped linking scores (official linking metric)
+    typeF/- bootstrapped argument F scores broken down by type (diagnostic)
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: type --> event type (note: EAL event type encompasses both type and subtype in RichERE)

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    typeRoleF/- bootstrapped argument F scores broken down by type and role (diagnostic)
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: type --> event type

File: tac-kbp-eal-scorer/src/main/resources/README.scorer.txt [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    mentionTypeF/ - bootstrapped argument F scores broken down by
+                       argument mention type (diagnostic)
+    perDocument/ - contains argument and linking scores for each document
+                       as well as a log of all errors encountered in that document.
+    nonBootstrapped/ - contains argument and linking scores over the eval
```
Comment (marjorief) [inline tac-kbp-eal-scorer/src/main/resources/README.scorer.txt]: add --> This folder includes argument precision, recall, and F1 to supplement the ArgScore.

**Pull request merged**